var/spack/repos/builtin/packages/brahma/package.py

```diff
@@ -15,20 +15,31 @@ class Brahma(CMakePackage):
 
     license("MIT")
 
-    version("develop", branch="dev")
+    version("develop", branch="develop")
     version("master", branch="master")
+    version("0.0.5", tag="v0.0.5", commit="219198c653cc4add845a644872e7b963a8de0fe2")
+    version("0.0.4", tag="v0.0.4", commit="8f41cc885dd8e31a1f134cbbcbaaab7e5d84331e")
     version("0.0.3", tag="v0.0.3", commit="fd201c653e8fa00d4ba6197a56a513f740e3014e")
     version("0.0.2", tag="v0.0.2", commit="4a36d5c08787d41c939fa1b987344b69d9ef97a6")
     version("0.0.1", tag="v0.0.1", commit="15156036f14e36511dfc3f3751dc953540526a2b")
 
     variant("mpi", default=False, description="Enable MPI support")
     depends_on("cpp-logger@0.0.1", when="@:0.0.1")
-    depends_on("cpp-logger@0.0.2:", when="@0.0.2:")
+    depends_on("cpp-logger@0.0.2", when="@0.0.2:")
```

Comment:
Do you want to end the range here?

```suggestion
    depends_on("cpp-logger@0.0.2", when="@0.0.2:0.0.3")
```